README.md

```diff
@@ -99,6 +99,8 @@ The new middlewares (`clerk.WithSessionV2()` & `clerk.RequireSessionV2()`) also
 - clerk.WithLeeway() to set a custom leeway that gives some extra time to the token to accommodate for clock skew
 - clerk.WithJWTVerificationKey() to set the JWK to use for verifying tokens without the need to fetch or cache any JWKs at runtime
 - clerk.WithCustomClaims() to pass a type (e.g. struct), which will be populated with the token claims based on json tags.
+- clerk.WithSatelliteDomain() to skip Issuer verification.
```

Comment:
:upside_down_face: Perhaps add more info on what `Issuer` means here? It's the "iss" JWT claim for the token.

Reply:
You are right, term out of context.